src/components/forms/PaymentOptionsForm/PaymentOptionsForm.tsx

```diff
@@ -423,7 +429,7 @@ function PaymentOptionsForm({
             className={`flex flex-col data-[state=active]:p-6 data-[state=active]:border border-dark-grey rounded h-full data-[state=inactive]:size-0 data-[state=inactive]:opacity-0 data-[state=active]:min-h-[405px]`}
           >
             <CardPanel
-              setIsValid={setIsValid}
+              setIsValid={paymentMethod === 'card' ? setIsValid : () => null}
```

Comment:
nit: add this logic to to useEffect vs here. 


i.e. if paymentMethod === "card" setIsValid(true)

Reply:
I think lets leave that for now - the organization of the callback is a bit wierd because setIsValid is passed into CardPanel and the validation there is handled with callbacks from internal components, but theres no panel components for the other methods. So we dont want to always isValid to true on that panel.